tests/cpp/mgclient_value.cpp

```diff
@@ -0,0 +1,324 @@
+#include <unistd.h>
```

Comment:
This is a `unit` test, so move it to the `unit` folder (create it).

Reply:
What about the rest of the tests like allocator, decoder, encoder...? Can't they also be classified as unit tests?

Reply:
Yes, they are also unit tests, but you can move them in another diff.